jenkins-pipelines/perf-regression-latency-nosqlbench-mixed.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
```

Comment:
The name of this pipeline should include the configuration name that it's using. 
Otherwise how can we differ them?

Reply:
Eventually, I'm expecting to see something like 6 pipelines:
num_of_tests (2) * num_of_configurations (3) = 6 total pipeline jobs.

Everytime someone adds a configuration, he needs to add the needed pipelines for it.

Reply:
Fixed (additional pipeline jobs added and name updated to reflect the different test case x configuration combination)